.github/ISSUE_TEMPLATE/feature.yml

```diff
@@ -53,6 +53,8 @@ body:
     - SDK
     - Wing Console
     - IDE Extension
+    - CLI
+    - Plugin
```

Comment:
What is "plugin"?

Reply:
Wing Plugin System. E.g. for this issue: [Enable Plugins to Explicitly State Compatibility #1474](https://github.com/winglang/wing/issues/1474)

Reply:
So maybe "Plugins"?

🤷🏻‍♀️

Reply:
ok, done